package.json

```diff
@@ -17,7 +17,9 @@
     "css:stats": "analyze-css --file build/bitstyles.css --pretty > test/css-stats.json",
     "precss:graph": "yarn build",
     "css:graph": "specificity-graph build/bitstyles.css --output test/specificity_graph/",
-    "build-storybook": "build-storybook"
+    "build-storybook": "yarn build && build-storybook",
```

Comment:
Is this overriding the storybook-internal `build-storybook` command? Wonder why this does not collide 💥

Reply:
Hmm, the name of this script was already `build-storybook`, as setup by storybook itself on install. I guess it doesn’t clash because one is run with yarn/npm, the other is a CLI tool?

Reply:
Oh, makes sense 🙈 

Thanks for explaining 🙇